scripts/release_issue_status/main.py

```diff
@@ -124,9 +139,18 @@ def main():
             item.bot_advice = 'new issue and better to confirm quickly.'
         elif item.delay_from_latest_update >= 7:
             item.bot_advice = 'delay for a long time and better to handle now.'
-        elif item.delay_from_create_date >= 30:
-            item.bot_advice = 'delay for a month and better to close.'
-
+  
+        if item.delay_from_create_date >= 30 and item.language == 'Python' and not item.whether_author_comment and '30days attention' not in item.labels:
+            item.labels.append('30days attention')
+            item.issue_object.set_labels(*item.labels)
+            issue.issue_object.edit(state='close')
```

Comment:
```suggestion
            item.issue_object.create_comment('hi @{} , the issue is closed since there is no reply. Please reopen it if necessary or create new,'
                                             ' '.format(item.author))
            issue.issue_object.edit(state='close')
```